### DISCUSSION TITLE
Using rdkit for Maven project

### DISCUSSION THREAD

**Discussion by tnguy271** [rdkit/rdkit#5608] [Q&A]
Does anyone have experience with using Java wrapper class for RDKit in Maven project? I was wondering how to incorporate the jar files and native library (.jnilib, .so, and .dll) to Maven project structure. I'm using Intellij IDEA. Appreciate any help.

**Comment by greglandrum** [comment]
You might take a look at the neo4j integration, I believe that uses maven: https://github.com/rdkit/neo4j-rdkit
